plugins/extractors/bigquery/README.md

```diff
@@ -60,17 +60,33 @@ source:
+| `schema`                       | [][Column](#column)                                                                                                                                                                                                                                                                                                                                                                         |                                                         |
+| `properties.partition_data`    | `"partition_data": {"partition_field": "data_date", "require_partition_filter": false, "time_partition": {"partition_by": "DAY","partition_expire": 0 } }`                                                                                                                                                                                                                                  | partition related data for time and range partitioning. |
+| `properties.clustering_fields` | `['created_at', 'updated_at']`                                                                                                                                                                                                                                                                                                                                                              | list of fields on which the table is clustered          |
+| `properties.partition_field`   | `created_at`                                                                                                                                                                                                                                                                                                                                                                                | returns the field on which table is time partitioned    |
+
+### Partition Data
+
+| Field                                     | Sample Value | Description                                                                                                                                                                                         |
+|:------------------------------------------|:-------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `partition_field`                         | `created_at` | field on which the table is partitioned either by TimePartitioning or RangePartitioning. In case field is empty for TimePartitioning _PARTITIONTIME is returned instead of empty.                   |
+| `require_partition_filter`                | `true/false` | returns the top level value set at bigquery table for RequirePartition                                                                                                                              |
```

Comment:
Can we change this description to - 

boolean value which denotes if every query on the bigquery table must include at least one predicate that only references the partitioning column. See [require-filter](https://cloud.google.com/bigquery/docs/managing-partitioned-tables#require-filter) for reference

Reply:
Done